src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    public static final double PI = 3.14159265358979323846;
+    private int radius;
+
+    public Circle() {
+        super();
+        this.radius = new Random().nextInt(100);
```

Comment:
According to the checklist, you should generate random values in `FigureSupplier` rather than inside the figure classes. Consider moving the random radius generation to `FigureSupplier` and passing it to the constructor.